src/app/hero-detail/hero-detail.component.ts

```diff
@@ -59,7 +59,6 @@ export class HeroDetailComponent implements OnInit {
           },
         }
       )
-      .pipe(take(1))
```

Comment:
That's the default behavior of mutations, they emit the result once.

Reply:
I was going to ask about that. Good to know!